.github/workflows/go.yml

```diff
@@ -0,0 +1,27 @@
+name: Go
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+
```

Comment:
Please remove this empty line.

Reply:
lol, yeah, github propagated this format by default, so I went with it... lol. changing it,